src/goto-symex/symex_main.cpp

```diff
@@ -25,6 +25,45 @@
 #include <util/std_expr.h>
 #include <vector>
 
+bool goto_symext::check_incremental(
+  expr2tc &expr,
```

Comment:
expr can be const here: when calling assertion and assume, you can use gen_false_expr() directly.

Reply:
Fixed.